components/ActionNetworkForm.vue

```diff
@@ -152,6 +154,10 @@ export default {
     }
   },
 
+  created() {
+    this.comment = this.$store.state.letterText
```

Comment:
This makes sense, but I worry that we're going to forget to change the letterText on some campaign that doesn't have a visible comment field and send a bunch of lorem ipsum text to Action Network. Might be best to default that to an empty string in config.json.